plugins/BEdita/Core/tests/TestCase/Model/Table/ObjectTypesTableTest.php

```diff
@@ -492,4 +499,36 @@ public function testFindAll()
         static::assertArrayHasKey('LeftRelations', $contain);
         static::assertArrayHasKey('RightRelations', $contain);
     }
+
+    /**
+     * Test default parent.
+     *
+     * @return void
+     *
+     * @covers ::beforeSave()
+     */
+    public function testParent()
+    {
+        $data = [
+            'singular' => 'foo',
+            'name' => 'foos',
+            'plugin' => 'BEdita/Core',
+            'model' => 'Objects',
+        ];
+
+        $objectType = $this->ObjectTypes->newEntity();
+        $this->ObjectTypes->patchEntity($objectType, $data);
+
+        $success = $this->ObjectTypes->save($objectType);
+        static::assertTrue((bool)$success);
+
+        $this->ObjectTypes->delete($objectType);
+        $data['parent_name'] = 'objects';
+        $this->ObjectTypes->patchEntity($objectType, $data);
+
+        $success = $this->ObjectTypes->save($objectType);
+        static::assertTrue((bool)$success);
+
+        $this->ObjectTypes->delete($objectType);
```

Comment:
This instruction is not needed: fixtures already take care of cleaning up things.